src/pages/ExperienceBeginning.svelte

```diff
@@ -1 +1,14 @@
-<h2>Start the experience</h2>
+<section>
+    <h2>{$_('experience_beginning.experiment_will_begin')}</h2>
+
+    <p>{$_('experience_beginning.start_if_you_are_ready')}</p>
+
+    <NextButton>
+        🠗 {$_('next')} 🠗
```

Comment:
The icons are not really necessary IMHO. In case we need those please use the one from bootstrap icons. There is an example here https://github.com/Lyokolux/ux-typing-experiment/blob/develop/src/components/LocaleSelect.svelte#L3

Reply:
YAGNI for now as we don't have clearly defined design, it will come after as a simple button is enough for now.